src/universal/redux/makeReducer.js

```diff
@@ -6,15 +6,27 @@ import authToken from './authDuck';
 import {reducer as storageReducer} from 'redux-storage-whitelist-fn';
 import storageMerger from 'universal/redux/storageMerger';
 
-const currentReducers = {
+const initialReducers = {
   authToken,
   cashay: cashayReducer,
   form: formReducer,
   notifications
 };
+const currentReducers = {...initialReducers};
 
 export default (newReducers) => {
   Object.assign(currentReducers, newReducers);
```

Comment:
Possible fix, I think we may have to keep extending `initialReducers` as well with the initial states of any new reducers we've loaded. What do you think @mattkrick?